packages/fiori-generator-shared/src/installedCheck.ts

```diff
@@ -0,0 +1,22 @@
+import { coerce, lt } from 'semver';
+
+/**
+ * Check for an installed extension, optionally specifying a minimum version.
+ * Note, this does not check for activation state of specified extension.
+ *
+ * @param vscode - vscode instance
+ * @param extensionId - the id of the extension to find
+ * @param minVersion - the minimum version of the specified extension, lower versions will not be returned. Must be a valid SemVer string.
+ * @returns true if the extension is installed and the version is >= minVersion (if provided), false otherwise
+ */
+export function isExtensionInstalled(vscode: any, extensionId: string, minVersion?: string): boolean {
```

Comment:
It will be need in the reference libary generator, I added it while I was in here. And I can also make the updates when consuming back into tool-suite